Core/WebAPI/Helpers/JwtTokenHelper.cs

```diff
@@ -5,83 +5,77 @@
 
 namespace Humteria.WebAPI.Helpers;
 
-public class JwtTokenHelper
+public interface IJwtGenerator
 {
-    private static readonly IConfiguration _configuration;
-    private static readonly string _TokenSecret;
-    private static readonly string _Audience;
-    private static readonly string _Issuer;
+    public string GenerateToken(JWTUserForTokenDTO user, int daysToExpire = 5);
+    public Guid ValidateToken(string token);
+}
 
+public class JwtTokenHelper : IJwtGenerator
+{
+    private readonly string m_tokenSecret;
```

Comment:
same here with m_

Reply:
it's just a habit haha
I add a `m_` prefix for member fields and `s_` for static fields. I guess we can just use the underscore if you've used that everywhere else, I haven't really looked closer into it when writing the code.

Reply:
Didn't see that being done, so I was just confused. Maybe a issue to do it, as I need to read into it.
Creating a discussion based on this